### PR TITLE
ignore the pycache files

### PR DESCRIPTION
**Pull request by Tvde1**

### CODE REVIEW

**Comment by torzdf** [comment]
I'm confused. pycache is already ignored:

image

**Comment by Tvde1** [comment]
Well my git client showed the temporary files as unstaged. It's also not in the `.gitignore` yet.

**Comment by AbysmalBiscuit** [comment]
@torzdf I think line 22 in the `.gitignore` file may un-ignore the nested __pycache__.

Perhaps changing the commit such that it would apply to all __pycache__ directories would be better, i.e. as suggested [here](https://github.com/github/gitignore/blob/master/Python.gitignore):
```python
__pycache__/
```

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,5 @@
 !plugins
 !tools
 !tools/lib*
+
+tools/lib_alignments/__pycache__/
```
Comment (AbysmalBiscuit) [inline .gitignore]: Change this to:
```python
__pycache__/
```
In order to ignore all __pycache__ directories, no matter where they are.

**Pull request merged**